deploy/complete/helm-chart/mushop/charts/orders/templates/_helpers.tpl

```diff
@@ -94,6 +94,13 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
   readOnly: true
 {{- end -}}
 
+{{- define "orders.mount.init.wallet" -}}
+# for init container
+- name: wallet
+  mountPath: /usr/lib/oracle/19.3/client64/lib/network/admin/
```

Comment:
This is a workaround. We should review to fix the issue #138

Reply:
Yes - absolutely.